general/administration/building.md

```diff
@@ -12,29 +12,37 @@ Jellyfin supports several methods of building for different platforms and instru
 All package builds begin with these two steps:
 
 1. Clone the repository:
-    `git clone https://github.com/jellyfin/jellyfin.git`<br/>
-    `cd jellyfin`
+    ```sh
+    git clone https://github.com/jellyfin/jellyfin.git
+    cd jellyfin
+    ```
```

Comment:
What will the indentation look like in DocFX? Will code blocks even respect indentation?

Reply:
yes, they will. I fed i to `docfx` through a docker image and it did what I expected.

Reply:
![docfxindent](https://user-images.githubusercontent.com/463828/75114868-2cfc5080-565a-11ea-9b78-554abf3cad6e.png)

Reply:
Since we're making changes to these code blocks, can we add newlines before and after as well?